spec/controllers/blackouts_controller_spec.rb

```diff
@@ -223,6 +223,25 @@
     end
 
     context 'PUT update' do
+      shared_examples_for 'does not update blackout' do |attributes|
+        before do
+          put :update, id: FactoryGirl.create(:blackout), blackout: attributes
+          assigns(:blackout).reload
+        end
+
+        it { is_expected.to set_flash[:error] }
+        it { is_expected.to render_template(:edit) }
+        it 'should have attributes' do
+          expect(attributes).not_to be_nil
+        end
```

Comment:
this test is unnecessary -- you're testing that the shared example wasn't given nil, which you have control over